velero-plugins/pod/restore.go

```diff
@@ -32,6 +32,10 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	json.Unmarshal(itemMarshal, &pod)
 	p.Log.Infof("[pod-restore] pod: %s", pod.Name)
 
+	// ISSUE-61 : removing the node selectors from pods
+	// to avoid pod never getting `unscheduled` on destination
```

Comment:
Thinking this should read:

```
to avoid pod being `unschedulable` on destination
```

Reply:
dang! that's double negation. will make the change. thanks